src/App.tsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
-import axios from 'axios';
-import {BrowserRouter as Router, Switch, Route } from 'react-router-dom';
```

Comment:
Why are the two imports above this here in the first place? If you can get rid of these before you merge this PR that'd be ideal

Reply:
fixed